src/containers/costume-tab.jsx

```diff
@@ -78,6 +84,7 @@ class CostumeTab extends React.Component {
 
         const addMessage = target.isStage ? addBackdropMsg : addCostumeMsg;
         const addFunc = target.isStage ? onNewBackdropClick : onNewCostumeClick;
+        const costume = this.props.vm.editingTarget.sprite.costumes[this.state.selectedCostumeIndex];
```

Comment:
This will error when `editingTarget` is null (a state I believe occurs when the app first loads), and error if the sprite costumes list hasn't been initialized (that is the reason for the ternary `target.costumes ?` check and the `target.costumes || []`). You could change this to `target.costumes[this.state.selectedCostumeIndex].rotationCenterX` below inside PaintEditor pros.